admin/tests/acceptance/authenticated/complementary-certifications/complementary-certification/attach-target-profile_test.js

```diff
@@ -99,8 +99,68 @@ module(
             });
           });
         });
+
+        module('when user selects an attachable target profile', function () {
+          test('it should display the link of the selected target profile with a change button', async function (assert) {
+            // given
+            await authenticateAdminMemberWithRole({ isSuperAdmin: true })(server);
+            server.create('complementary-certification', {
+              id: 1,
+              key: 'KEY',
+              label: 'MARIANNE CERTIF',
+              targetProfilesHistory: [{ name: 'ALEX TARGET', id: 3, attachedAt: dayjs('2023-10-10T10:50:00Z') }],
+            });
+            server.create('attachable-target-profile', {
+              id: 3,
+              name: 'ALEX TARGET',
+            });
+            const screen = await visit('/complementary-certifications/1/attach-target-profile/3');
+            const input = screen.getByRole('searchbox', { name: 'ID du profil cible' });
+            await fillIn(input, '3');
+            await waitFor(async () => {
+              await screen.findByRole('listbox');
+            });
```

Comment:
En principe les `findBy` sont du coup déjà un `waitFor` + `query` (ici + dans les autres tests)

```suggestion
           await screen.findByRole('listbox');
```

Source : https://testing-library.com/docs/dom-testing-library/api-async/

Reply:
Effectivement j'ai pu enlever tous les `waitfor` des tests et les remplacer par des findByRole 👌